static/css/components/tagging-menu.less

```
@@ -245,6 +245,18 @@
       background-color: @primary-blue;
     }
   }
+
+  &.highlight-none-tagged {
+    background-color: @light-grey;
+  }
+
+  &.highlight-some-tagged {
+    background-color: @light-yellow;
+  }
+
+  &.highlight-all-tagged {
+    background-color: @light-yellow;
+  }
```

```suggestion
  &--staged {
    background-color: @light-yellow;
  }
```
These menu options will either be staged or not.  Let's use `.selected-tag--staged` to add the yellow background to menu options.